tests/garage/tf/models/test_cnn_model.py

```diff
@@ -61,15 +62,18 @@ def test_output_value(self, filter_sizes, in_channels, out_channels,
 
         assert np.array_equal(output, expected_output)
 
-    @params(((1, ), (3, ), (32, ), (1, ), (1, 1), (1, 1)),        # noqa: E501, yapf: disable
-            ((3, ), (3, ), (32, ), (1, ), (2, 2), (1, 1)),        # noqa: E501, yapf: disable
-            ((3, ), (3, ), (32, ), (1, ), (1, 1), (2, 2)),        # noqa: E501, yapf: disable
-            ((3, ), (3, ), (32, ), (1, ), (2, 2), (2, 2)),        # noqa: E501, yapf: disable
-            ((3, ), (3, ), (32, ), (2, ), (1, 1), (2, 2)),        # noqa: E501, yapf: disable
-            ((3, ), (3, ), (32, ), (2, ), (2, 2), (2, 2)),        # noqa: E501, yapf: disable
-            ((1, 1), (3, 32), (32, 64), (1, 1), (1, 1), (1, 1)),  # noqa: E501, yapf: disable
-            ((3, 3), (3, 32), (32, 64), (1, 1), (1, 1), (1, 1)),  # noqa: E501, yapf: disable
-            ((3, 3), (3, 32), (32, 64), (2, 2), (1, 1), (1, 1)))  # noqa: E501, yapf: disable
+    @pytest.mark.parametrize('filter_sizes, in_channels, out_channels, '
+                             'strides, pool_strides, pool_shapes', [
+        ((1,), (3,), (32,), (1,), (1, 1), (1, 1)),  # noqa: E501, yapf: disable
```

Comment:
none of these lines are over length. you should be able to yapf: disable the entire block.